blog/2021-08-19-apisix-casbin-authorization.md

```diff
@@ -47,7 +47,7 @@ m = (g(r.sub, p.sub) || keyMatch(r.sub, p.sub)) && keyMatch(r.obj, p.obj) && key
 
 From the above scenario, the policy would be:
 
-```conf
+```csv
```

Comment:
The `csv` tag doesn't give colorzing. Is there any way to improve the format for CSV policy? Like another tag or even write our own formatting plugin for a new tag.

Reply:
This is a problem. Prism plugin for `csv` is not yielding the results we want. I guess this calls for some custom code.

Reply:
OK